src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,399 @@
+
+  constructor(uuid: string, amount?: number) { // Can be built with a given amount, but if not will seed the number of jobs to build
+    this.uuid = uuid;
+    this.seed = Math.abs(this.convertStrToNum(uuid));
+    
+    this.jobs = [];
+    // generate a random number of jobs, weighted towards ~3
+    const generatedJobCount = Math.round( (this.nextRand()*7 + this.nextRand()*7) / 2 );  // This line needs to run regardless of if we were passed an amount, or else the seeded rng will get messed up!
+    this.jobCount = amount ? amount : generatedJobCount; // allows for front end to request a specific number of jobs. If amount is not present, use the above generated number
+    for (let i = 0; i < this.jobCount; i++) {
+      this.jobs.push(this.generateJob()); // Where all the "magic" happens, generateJob will create all job data and push it into job array
+    }
+  }
+
+  /**
+   * @description - Called by constructor to generate a job. 
+   * @returns {JobData} - Returns an object which contains the entirety of job data required to fill out a fake ticket
+   */
+  private generateJob(): JobData {
+    // The following variables are generated prior to the output object, as they are passed into generator methods in order to generate data consistent throughout the ticket
+    const firstName = this.generateFirstName();
+    const lastName = this.generateLastName();
+    const streetAddress = this.generateStreetAddress();
+    const jobType = this.randOf(['Install', 'Repair', 'Helper', 'POTS', 'BSW']);
+    const transportType = jobType === 'POTS' || this.nextRand() < 0.5 ? this.randOf(['IP-CO', 'IP-RT', 'FTTN', 'FTTN-BP']) : this.randOf(['FTTP GPON', 'FTTP XGS-PON']);
+    const curJob: JobData = {
+      firstName: firstName,
+      lastName: lastName,
+      accountNumber: this.randDigits(7),
+      location: {
+        zip: this.randDigits(5),
+        ...this.generateCityLocation(streetAddress)
+      },
+      appointment: this.generateAppointment(),
+      email: this.generateEmail(firstName, lastName),
+      phone: this.randDigits(10),
+      history: this.generateHistory(),
+      jobType: jobType,
+      transportType: transportType,
+      facilities: this.generateFacilities(transportType, streetAddress),
+      services: this.generateServices(transportType)
+    };
+
+    return curJob;
+  }
+
+  /**
+   * @description Found online. Converts a string to a unique number for seed generation.
+   * @param {string} str The string to convert
+   * @returns {number} the string converted to a unique number
+   */
+  private convertStrToNum(str: string): number {
+    let hash = 0;
+    for (let i = 0; i < str.length; i++) {
+      hash = ((hash << 5) - hash) + str.charCodeAt(i); // essentially convert the number to binary, shift the bits over 5, and add the new number
+      hash |= 0; // converts the number to a 32bit integer
```

Comment:
Not familiar with `|=`

Reply:
I had to look it up, because that whole dealio there I found online. It is a "bitwise or operator". So if you have two binary numbers, say 26 and 12, in binary they would be
`26 -> 11010`
`12 -> 01100`

So if we do `const result = 26 | 12` it would compare each bit of the above numbers and this would be the `result`:
`11010`
`01100`
`-----`
`11110`
So start with the ones column. Are 0 OR 0 "true"? No, so the result is 0. Then the tens column. Are 1 OR 0 "true"? Yes, so the result is 1. and so on. The answer 30.

The above thing, `num |= 0` essentially allows you convert whatever wild number is stored in has back into an integer. Don't fully understand how it works but I'm guessing it's more efficient than whatever pleb way we would do it.

Reply:
I'll have to check our code but I don't think we use it. 

Typically the use of helpful operators like this has to be agreed upon and put out as new standard. 

Add a comment above it showing an alternate approach.

Reply:
I just removed it. I don't know why we would need a 32 bit integer, and also I'm pretty sure the strings we're using won't be anywhere near 32 bits when converted to a number. I understand how the algorithm works, and this isn't needed.